### PR TITLE
Add coolhigh canbus code which is cherry-picked from v3.1_dev to branch master

### PR DESCRIPTION
**Pull request by lishuanghua**
Add coolhigh canbus code which is cherry-picked from v3.1_dev to branch master

### CODE REVIEW

File: modules/canbus/vehicle/ch/ch_controller.cc [hunk 1]
```diff
@@ -0,0 +1,556 @@
+/* Copyright 2019 The Apollo Authors. All Rights Reserved.
```
Comment (xiaoxq) [inline modules/canbus/vehicle/ch/ch_controller.cc]: Unify copyright format.

File: modules/canbus/vehicle/ch/ch_message_manager.cc [hunk 2]
```diff
@@ -0,0 +1,60 @@
+/* Copyright 2019 The Apollo Authors. All Rights Reserved.
```
Comment (xiaoxq) [inline modules/canbus/vehicle/ch/ch_message_manager.cc]: ditto

File: modules/canbus/vehicle/ch/ch_controller.cc [hunk 3]
```diff
@@ -0,0 +1,556 @@
...
+  ResetProtocol();
+}
+
+ErrorCode ChController::EnableAutoMode() {
+  if (driving_mode() == Chassis::COMPLETE_AUTO_DRIVE) {
+    AINFO << "already in COMPLETE_AUTO_DRIVE mode";
+    return ErrorCode::OK;
+  }
+  // ADD YOUR OWN CAR CHASSIS OPERATION
+  // control_command_115_->set_ctrl_cmd(Control_command_115::CTRL_CMD_UNDER_CONTROL);
```
Comment (macDure) [inline modules/canbus/vehicle/ch/ch_controller.cc]: If this is not useful, please delete it.

File: modules/canbus/vehicle/ch/ch_controller.cc [hunk 4]
```diff
@@ -0,0 +1,556 @@
...
+    }
+
+    chassis_.set_gear_location(gear_pos);
+  } else {
+    chassis_.set_gear_location(Chassis::GEAR_NONE);
+  }
+  // 11 steering
+  if (chassis_detail.ch().has_steer_status__512() &&
+      chassis_detail.ch().steer_status__512().has_steer_angle_sts()) {
+    // * 100.0 /vehicle_params_.max_steer_angle()*M_PI / 180);
```
Comment (macDure) [inline modules/canbus/vehicle/ch/ch_controller.cc]: If this is not useful, please delete it.

**Pull request closed without merging**